src/encoded/static/components/viz/BarPlot/Chart.js

```diff
@@ -342,7 +342,7 @@ export class Chart extends React.PureComponent {
         barplot_color_cycler.resetCache();
 
         const {
-            width, height, showType, barplot_data_unfiltered, barplot_data_filtered,
+            width, height, showType, barplot_data_unfiltered, barplot_data_filtered, context,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['context' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)